packages/core/content-releases/admin/src/pages/ReleasesPage.tsx

```diff
@@ -43,8 +81,191 @@ const ReleasesPage = () => {
           </CheckPermissions>
         }
       />
+      {children}
+    </Main>
+  );
+};
+
+/* -------------------------------------------------------------------------------------------------
+ * ReleasesGrid
+ * -----------------------------------------------------------------------------------------------*/
+interface ReleasesGridProps {
+  sectionTitle: 'pending' | 'done';
+  releases?: GetReleases.Response['data'];
+  isError?: boolean;
+}
+
+const LinkCard = styled(Link)`
+  display: block;
+`;
+
+const ReleasesGrid = ({ sectionTitle, releases = [], isError = false }: ReleasesGridProps) => {
+  const { formatMessage } = useIntl();
+
+  if (isError) {
+    return (
+      <Flex paddingTop={8}>
+        <AnErrorOccurred />
+      </Flex>
+    );
+  }
+
+  if (releases?.length === 0) {
+    return (
+      <EmptyStateLayout
+        content={formatMessage(
+          {
+            id: 'content-releases.page.Releases.tab.emptyEntries',
+            defaultMessage: 'No releases',
+          },
+          {
+            target: sectionTitle,
+          }
+        )}
+        icon={<EmptyDocuments width="10rem" />}
+      />
+    );
+  }
+
+  return (
+    <Grid gap={4}>
+      {releases.map(({ id, name }) => (
+        <GridItem col={3} s={6} xs={12} key={id}>
+          <LinkCard href={`content-releases/${id}`} isExternal={false}>
+            <Flex
+              direction="column"
+              justifyContent="space-between"
+              padding={4}
+              hasRadius
+              background="neutral0"
+              shadow="tableShadow"
+              height="100%"
+              width="100%"
+              alignItems="start"
+            >
+              <Typography as="h3" variant="delta" fontWeight="bold">
+                {name}
+              </Typography>
+            </Flex>
+          </LinkCard>
+        </GridItem>
+      ))}
+    </Grid>
+  );
+};
+
+/* -------------------------------------------------------------------------------------------------
+ * ReleasesPage
+ * -----------------------------------------------------------------------------------------------*/
+const ReleasesPage = () => {
+  const [addReleaseDialogIsShown, setAddReleaseDialogIsShown] = React.useState(false);
+  const { formatMessage } = useIntl();
+  const [{ query }, setQuery] = useQueryParams<GetAllReleasesQueryParams>();
+  const response = useGetReleasesQuery(query);
```

Comment:
Looks like in your first request to the getAll endpoint you are sending the query empty, this means that you are returning both pending & done releases, only the first time you enter the releases page (when the url is `/content-releases`)

Probably we would want to change the releases sidebar link to `/content-releases?filters[released]....`

Reply:
No, I can set the default value in the getReleases endpoint inside RTK query to have the correct filters for the pending releases